docs/sources/enterprise/reporting.md

```diff
@@ -155,6 +155,12 @@ The page header of the report displays the time range for the dashboard's data q
 
 If the time zone is set differently between your Grafana server and its remote image renderer, then the time ranges in the report might be different between the page header and the time axes in the panels. We advise always setting the time zone to UTC for dashboards when using a remote renderer to avoid this.
 
+## Report template variables
```

Comment:
```suggestion
## Report-template variables
```
Rationale: Disambiguate grammar where report could otherwise be a verb.

Reply:
Should it be smth like `Report-specific template variables` maybe? `Report-template` sound pretty confusing, at least to me.

Reply:
How about `Variables of report templates`?

Reply:
`Template` here refers to dashboard templating, not report and in our docs we call [a query with a variable](https://grafana.com/docs/grafana/latest/variables/#templates) a template. So basically the functionality is about allowing users to select template variables for a dashboard from report.